### ISSUE TITLE
Ability to make minimum notice recognize Work Days 

### ISSUE THREAD

**Issue by eenmeier** [calcom/cal.com#15978]
### Is your proposal related to a problem? YES

I need to buffer out 2 days during the during workweek for event bookings. In order to allot for this on a Friday and not be booked on a Monday I have to make the min notice 4 days in my events. But during the week M-R I do not need 4 days min notice. 

### Describe the solution you'd like

Change the drop down under Limits -> Minimum Notice -> to include Work Days instead of just Days (which uses calendar days). Work Days vs Calendar Days is already an option under Limit Future Bookings. 

### Describe alternatives you've considered

I have to use 4 days now and that is too much M-R 

### Additional context

NO 
### Requirement/Document

No 

---
##### House rules
- If this issue has a `🚨 needs approval` label, don't start coding yet. Wait until a core member approves feature request by removing this label, then you can start coding.
  - For clarity: Non-core member issues automatically get the `🚨 needs approval` label.
  - Your feature ideas are invaluable to us! However, they undergo review to ensure alignment with the product's direction.
  - Follow Best Practices lined out in our [Contributor Docs](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)

**Comment by masteradhoc** [comment]
Would need this as well!

**Comment by eenmeier** [comment]
I do not feel this is low priority. @CarinaWolli Please re-evaluate.

**Comment by CarinaWolli** [comment]
I think it's a valuable feature, but it's currently a low priority for us. However, feel free to work on it, we'll be happy to review it 🙏

**Comment by buckspencer** [comment]
Would also greatly benefit from this
